StripePayments/StripePayments/Source/API Bindings/Models/Shared/STPIntentActionKonbiniDisplayDetails.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  STPIntentActionKonbiniDisplayDetails.swift
+//  StripePayments
+//
+//  Created by Yuki Tokuhiro on 9/12/23.
+//
+
+import Foundation
+
+/// Contains Konbini details necessary for the customer to complete the payment.
+public class STPIntentActionKonbiniDisplayDetails: NSObject, STPAPIResponseDecodable {
+    /// The date at which the pending Konbini payment expires.
+    @objc public let expiresAt: Date
```

Comment:
Why do we parse this? Is it used somewhere?

Reply:
It's not used but it's in the public API.